backend/pkg/scheduler.go

```diff
@@ -118,18 +119,57 @@ func createReports() {
 
 	qr, err := api.QueryFromResponse(response)
 
-	f, e := excelize.OpenFile("./data/test.xlsx")
+	// Open the template
+	f, e := excelize.OpenFile("./data/template.xlsx")
+
+	// Regexp which matches a cell reference [TODO: Handle $ refs]
+	reg := regexp.MustCompile("([A-Za-z]+)|([0-9]+)")
```

Comment:
Maybe missing something, but should the `|` be here? E.g. should a cell be `A3` (rather than `A` or `3`)?